lib/pronto/formatter/github_pull_request_review_formatter.rb

```diff
@@ -10,7 +10,7 @@ def pretty_name
       end
 
       def submit_comments(client, comments)
-        client.create_pull_request_review(comments)
+        client.publish_pull_request_comments(comments)
```

Comment:
@dimitrovv why limit this feature to GitHub? Just trying to understand :)

Reply:
yes, I thought about this, but I guess a review could only be submitted when using **Github** pronto client or I am wrong.

Reply:
👍